app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -122,6 +153,11 @@ class ParkingViewModel(
     _selectedParkingReports.value = emptyList()
   }
 
+  /**
+   * Adds a parking to the repository.
```

Comment:
Why is there the documentation twice ?

Reply:
I don't that is a great question a will remove it right away thanks for pointing it out !